src/services/API/Plio.js

```diff
@@ -130,6 +131,30 @@ export default {
     return resultSet.chartPivot()[0][resultKey];
   },
 
+  async getUniqueUsersCountList(plioIds) {
+    var resultSet = await analyticsAPIClient().load(
+      uniqueUsersListQuery(plioIds)
+    );
+
+    // holds the mapping of plio ID to count
+    var resultsMap = {};
+    if (resultSet.series()[0] != undefined)
+      resultSet.series()[0].series.forEach((seriesItem) => {
+        resultsMap[seriesItem.x] = seriesItem.value;
+      });
+
+    // holds the final list of values to be returned
+    var results = [];
+    plioIds.forEach((plioId) => {
+      // plios which do not have any sessions do not show up in
+      // the resultMap - use a default value for those plios
+      if (resultsMap[plioId] == undefined) results.push(0);
```

Comment:
```suggestion
      if (!(plioId in resultsMap)) results.push(0);
```
Seems more readable to me
There's no `not in` in javascript otherwise that would've been best

Reading the current code, it feels like its saying plioId key is present in resultsMap but the value of the key is undefined but actually plioId key doesn't exist in resultsMap.

I tested with this changes and it's working fine